hadoop-ozone/dist/src/main/smoketest/compatibility/read.robot

```diff
@@ -37,4 +37,12 @@ Dir Can Be Listed Using Shell
 File Can Be Get
     Execute    ozone fs -get o3fs://bucket1.vol1/dir-${SUFFIX}/file-${SUFFIX} /tmp/
     Execute    diff -q ${TESTFILE} /tmp/file-${SUFFIX}
-    [teardown]    Execute    rm /tmp/file-${SUFFIX}
+    [teardown]    Execute    rm -f /tmp/file-${SUFFIX}
+
+FSO Bucket Can Be Read
```

Comment:
Which "another test" do you mean?

Reply:
@ivanzlenko can you please clarify your comment?

Reply:
`read.robot` relies on the content created by `write.robot` by design, because we need to cross-check between versions.  E.g. running a 2.0.0 cluster write data with both 1.4.0 and 2.0.0 clients, then verify that they can read content written by the other.  This cannot be done in a single self-contained robot test, since that is run in a specific container with a specific Ozone version.

`upgrade` test is somewhat similar (write with one version, upgrade, write with new version, read both with new version).